audit/projects/k8s-staging-ci-images/services/enabled.txt

```diff
@@ -3,9 +3,7 @@ cloudbuild.googleapis.com         Cloud Build API
 cloudkms.googleapis.com           Cloud Key Management Service (KMS) API
 containerregistry.googleapis.com  Container Registry API
 logging.googleapis.com            Cloud Logging API
-monitoring.googleapis.com         Cloud Monitoring API
 pubsub.googleapis.com             Cloud Pub/Sub API
 secretmanager.googleapis.com      Secret Manager API
-stackdriver.googleapis.com        Stackdriver API
```

Comment:
This was me manually disabling these after I accidentally enabled them via the console (was in monitoring for another project that had it enabled, switched projects, since I am admin, the services auto enabled)